packages/core/src/session.ts

```diff
@@ -137,7 +137,7 @@ export function closeSession(session: Session, status?: Exclude<SessionStatus, '
  * @returns a JSON object of the passed session
  */
 function sessionToJSON(session: Session): SerializedSession {
-  return dropUndefinedKeys({
+  return {
```

Comment:
this is only called when we send the session, where this should be cleared out by `JSON.stringify()` anyhow